src/app/components/search-page/search-page.component.ts

```diff
@@ -1,15 +1,35 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { environment } from '../../../environments/environment';
+import { AuthService } from '../../services/auth.service'
 
 @Component({
   selector: 'app-search-page',
   templateUrl: './search-page.component.html',
   styleUrls: ['./search-page.component.scss']
 })
-export class SearchPageComponent implements OnInit {
+export class SearchPageComponent {
+  searchConfig = {
+    ...environment.algolia,
+    indexName: 'qwe'
+  }
 
-  constructor() { }
+  showResults = false;
 
-  ngOnInit() {
-  }
+  constructor(private authService: AuthService) { }
 
+  searchChanged(query) {
+    if (query.length) {
+      this.showResults = true;
+    } else {
+      this.showResults = false;
+    }
+  }
+  get status() {
+    if (this.authService.isLogged) {
+      return true
+    } else {
+      return false;
+    }
+  }
```

Comment:
I could not find the usage of this method. I assume you should delete all the unused code.